### DISCUSSION TITLE
Deployment to IIS website

### DISCUSSION THREAD

**Discussion by MarkStega** [wixtoolset/issues#7988] [Questions]
Is there an example of using WiX4 to deploy an IIS website? If there was a development dojo episode that would be great but certainly someone's blog would also work. Unfortunately, when I Google search there are tons of hits on the WiX web builder, not so much on WiX4.

**Comment by chrpai** [comment]
I have a FOSS project called IsWiX which has support for just this.  You can search for iswix-tutorials and you'll find examples for web.

Exactly what you need to do depends on what you are deploying to iis.  For example asp.net core will require also creating a bootstrapper to make sure that the hosting bundle is installed.

**Comment by MarkStega** [comment]
@chrpai 
Chris,

Thanks for the reference, I am working on understanding the installer now. I have a controlled set of environments so I know that the prerequisites (.Net 8 and IIS) are present so I can do a bit of simplification.

**Comment by robmen** [comment]
IIS configuration has not changed from WiX v3 to WiX v4. All the documentation from WiX v3 about IIS should still be relevant.
